pages/course/[course]/assignment/[assignment].tsx

```diff
@@ -130,7 +154,24 @@ const AssignmentView: NextPage<AssignmentProps> = ({ assignment }) => {
         </h1>
         <p>{assignment.description}</p>
         <h2 className="font-semibold text-2xl text-slate-50">Submissions</h2>
-        {assignment.submission.map(SubmissionCard)}
+          {assignment.submission.map(submission => {
+            if (typeof submission.student === "number") {
+              return null;
+            }
```

Comment:
how does this happen?

Reply:
I was confused on that myself. Didn't know `submission.student` could be `User | number | null` type. The compiler kept complaining if I didn't do some error checking.

Reply:
yeah i'd say dw about it for now, we can use the server client and rls later